packages/core/src/sessionManagement.ts

```diff
@@ -49,15 +56,23 @@ export function startSessionManagement<Type extends string>(
     }
   }, COOKIE_ACCESS_DELAY)
 
+  const expandSession = () => {
```

Comment:
This function could be in the scope where it's used (inside the `if`) or even inlined.

Reply:
By inlining it, we would lost the intention (renew session) and keeping the function instead of adding a comment make it more consistent with the `expandOrRenewSession` function.
When we will remove the if, it would end up at the same position as it is actually.
wdyt?

Reply:
All right